packages/atomic/src/components/insight/atomic-insight-search-box/atomic-insight-search-box.tsx

```diff
@@ -388,7 +388,10 @@ export class AtomicInsightSearchBox {
   public render() {
     return (
       <SearchBoxWrapper disabled={this.disableSearch}>
-        <atomic-focus-detector onFocusExit={() => this.clearSuggestions()}>
+        <atomic-focus-detector
+          style={{display: 'contents'}}
```

Comment:
Can we remove the `display: contents` applied inside `atomic-focus-detector` then ?

Reply:
I don't know... I would keep it as a fallback in case someone forgets to add the property at this level.